### ISSUE TITLE
[Enhancement] Including distrobox in SUSE repository

### ISSUE THREAD

**Issue by 89luca89** [89luca89/distrobox#155]
Installing distrobox is pretty simple, but it would be nice to install it from a distribution's repository.

I have no experience as a maintainer or packager, so if anyone wants to give a shot at this, it's welcome :smile:

It could be of help that an RPM Spec is available here: https://koji.fedoraproject.org/koji/buildinfo?buildID=1902894

**Comment by dfaggioli** [comment]
Indeed. In fact, there are packages available already. E.g., this is mine (on top of the one from critbase [1], which did it a few days ago): https://build.opensuse.org/package/show/home:dfaggioli:microos-desktop/distrobox .

You can already use it like this, if you want:

```
zypper addrepo https://download.opensuse.org/repositories/home:dfaggioli:microos-desktop/openSUSE_Tumbleweed/home:dfaggioli:microos-desktop.repo
zypper refresh
zypper install distrobox
```

I agree that it would be cool for it to just be available in openSUSE Tumbleweed official repos, and I'm already working on getting it there.

[1] https://build.opensuse.org/package/show/home:critbase/distrobox

**Comment by dfaggioli** [comment]
In the meantime, I'll put some info in the host distro compatibility page. PR coming soon :-)

**Comment by dfaggioli** [comment]
> In the meantime, I'll put some info in the host distro compatibility page. PR coming soon :-)
>
https://github.com/89luca89/distrobox/pull/157

**Comment by 89luca89** [comment]
That's great thanks @dfaggioli :smile:

**Comment by 89luca89** [comment]
Can I help in any way to ease the package approval?

**Comment by 89luca89** [comment]
Heads up, next version will include #190 
This will DEPRECATE the use of `--path/-p` in favor of `--prefix/-P` to behave similar to makefiles.
This way you specify only the base path where `bin` and `man` will be installed, for example

`./install --prefix /usr/local`

**Comment by 89luca89** [comment]
update: 

as of version `1.2.14` it is possible to specify defaults in a configuration file in the following system locations:

- /usr/share/distrobox/distrobox.conf
- /etc/distrobox/distrobox.conf

This will allow a bit of personalization during packaging, for example it would be possible for ubuntu to specify:

```sh
container_image="registry.opensuse.org/opensuse/toolbox:latest"
container_manager="autodetect"
container_name="opensuse-distrobox"
```

**Comment by fcrozat** [comment]
distrobox is now available in openSUSE Factory see https://build.opensuse.org/package/show/openSUSE:Factory/distrobox

I think you can close this issue ;)
